nw/core/index.py

```diff
@@ -356,7 +356,7 @@ def scanText(self, tHandle, theText):
 
         # Count words for remaining text after last heading
         if nTitle > 0:
-            lastText = "\n".join(theLines[nTitle-1:nLine-1])
+            lastText = "\n".join(theLines[nTitle-1:])
```

Comment:
This one caused the section word count of the last line of a file to be missing.